.github/container/test-pax.sh

```diff
@@ -55,6 +57,8 @@ NVTE_FUSED_ATTN=1
 DROPOUT=0
 EVALUATE=0
 ADDITIONAL_ARGS=""
+ENABLE_FMHA=${ENABLE_FMHA:-0}
```

Comment:
Sorry, one more comment. Can we enable FMHA by default? We recently enabled FMHA by default in our nightly images, so it would be good to keep the same behavior here

Reply:
Enabling it here by default will mean that for all the pax tests the fmha will be enabled, do we want that?

Reply:
Yes, that's the current behavior. Only this 5b test runs with FMHA disabled: https://github.com/NVIDIA/JAX-Toolbox/blob/bcb72054e9b3e707a31d59e4a68fcc2d7fa8a96f/.github/workflows/_test_pax_rosetta.yaml#L255

Reply:
Enabled by default now, updated args for 5b model in _test_pax_rosetta.yaml